CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.4.2] - 2023-11-29
+### Fixed
+- Update `Workflow Version` upsert docs to reflect that `upsert` is called on the `version` subresource
+
```

Comment:
We dont need a new release to push new docs
```suggestion
```

Reply:
Good to know!